src/silo/storage/sequence_store.test.cpp

```diff
@@ -0,0 +1,47 @@
+#include "silo/storage/sequence_store.h"
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <string>
+
+std::unique_ptr<silo::SequenceStore> setupSequenceStore() {
+   std::unique_ptr<silo::SequenceStore> sequence_store = std::make_unique<silo::SequenceStore>();
+
+   // Construct test genomes.
+   std::string test_genome1 = "A";
+   test_genome1.reserve(silo::GENOME_LENGTH);
+   for (unsigned i = 1; i < silo::GENOME_LENGTH; ++i) {
+      test_genome1.push_back('A');
+   }
+
+   std::string test_genome2 = "C";
```

Comment:
I think, we can use this also in other places, so I would suggest making it a function, and therefore getting rid of the comment as well.

Reply:
Good idea!